### PR TITLE
Cleanup and restructure API reference overview

### PR DESCRIPTION
**Pull request by morphis**
This drops things which do not belong in there / or do not belong together.

The dropped API overview for AMS will move into the swagger where it belongs.

### CODE REVIEW

**Comment by keirthana** [comment]
> The dropped API overview for AMS will move into the swagger where it belongs.

@jat-canonical @morphis Are there PRs or Jira issues tracking the overview content move to swagger. I am asking so that we don't lose track of this once I merge this PR

File: reference/api-reference.md
Comment (keirthana) [inline reference/api-reference.md:4]:
```suggestion
Anbox Cloud includes various APIs to help you build solutions with it. This page provides an overview of the available APIs and their uses.
```

File: reference/api-reference.md
Comment (keirthana) [inline reference/api-reference.md:12]: vowel sound, so an HTTP
```suggestion
The AMS services provide an HTTP based API to help you manage all of its objects. Internally the `amc` client uses the same API to perform its operation.
```

File: reference/api-reference.md
Comment (keirthana) [inline reference/api-reference.md:18]:
```suggestion
The Anbox runtime inside each instance managed by the AMS provides an HTTP API over a UNIX domain socket that helps interacting with various functionalities offered by Anbox. This includes the injection of position or sensor events into Android, among other things.
```

File: reference/api-reference.md
Comment (keirthana) [inline reference/api-reference.md:24]:
```suggestion
The Anbox Stream Gateway service provides an API endpoint to help you to manage streaming sessions.
```

**Review by keirthana — CHANGES_REQUESTED** [review]
Minor comments based on the Ubuntu style guide and one question about moving the overview to swagger.
Otherwise, LGTM!

**Comment by morphis** [comment]
@keirthana yes, see https://github.com/canonical/ams/pull/495

**Review by keirthana — APPROVED** [review]
LGTM, if spellcheck issues are addressed (@morphis)

**Pull request merged**
